lectures/inflation_history.md

```diff
@@ -10,24 +10,23 @@ kernelspec:
   language: python
   name: python3
 ---
-
-# 价格水平历史 
+# 价格水平历史
 
 本讲将讨论一些关于综合价格指数波动的历史数据。
 
 我们首先安装必要的Python包。
 
-`xlrd` 在这里被引入是因为 `pandas` 需要它来对Excel文件执行操作。
+这里引入 `xlrd`是因为 `pandas` 需要它来对Excel文件执行操作。
 
-```{code-cell} ipython3
+```{code-cell}
 :tags: [hide-output]
 
 !pip install xlrd
 ```
 
 <!-- Check for pandas>=2.1.4 for Google Collab Compat -->
 
-```{code-cell} ipython3
```

Comment:
Thanks @yuki-820,

Please keep the `ipython3` in the code cell since it is used to determine the type of the code. I will revert these changes.